config/log/discard.go

```diff
@@ -0,0 +1,17 @@
+package log
+
+func NewDiscard() Log {
```

Comment:
similar question, we could use the common discard logger https://github.com/anchore/go-logger/blob/main/adapter/discard/logger.go